content/en/synthetics/guide/browser_tests_without_chrome_extension.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Datadog Browser tests without the Chrome extenstion
+kind: guide
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-synthetic-monitoring/'
+      tag: 'Blog'
+      text: 'Introducing Datadog Synthetics'
```

Comment:
This blogpost is mostly about API tests, could we use that one https://www.datadoghq.com/blog/browser-tests/ instead?
Do you think we could add the browser test getting started guide as well?